src/processor.js

```diff
@@ -74,6 +75,7 @@ module.exports = class Processor {
         vm,
         options,
         extraData,
+        smtLevels,
```

Comment:
- missing parameter in the comments
- is this a required parameter? could we add in in the `options` object ?
  - it has no default value, is it ok ?
  - throw error if parameter is `undefined`

Reply:
Indeed, this parameter is redundant as it is already embbeded at the smt instance